src/Plugin/QueueWorker/HubspotCommerceQueue.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\hubspot_commerce\Plugin\QueueWorker;
+
+use Drupal\hubspot_commerce\Services\HubspotCommerce;
+
+use Drupal\Core\Annotation\QueueWorker;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class HubspotCommerceQueue.
+ *
+ * @QueueWorker(
+ *   id = "hubspot_commerce_queue",
+ *   title = @Translation("Hubspot Commerce queue"),
```

Comment:
**CHANGE REQUEST**

The ID or the title says nothing about what the queue worker does. The `queue` suffix is also unnecessary, it's a queue worker so it obviously process queue items.

Let's have the ID be `commerce_hubspot_entity_sync_to` and the title `Sync Drupal entities to HubSpot`. The `sync_to` is to allow for a `sync_from` worker in case we later need to implement syncing back when entities are changed in HubSpot.